tests/polkadotjs_test/start_polkadotjs_test.go

```diff
@@ -30,13 +30,23 @@ func TestStartGossamerAndPolkadotAPI(t *testing.T) {
 	nodes, err := utils.InitializeAndStartNodesWebsocket(t, 1, utils.GenesisDev, utils.ConfigDefault)
 	require.NoError(t, err)
 
-	command := "npx mocha ./test --timeout 30000"
+	node := nodes[0]
+
+	const command = "npx mocha ./test --timeout 30000"
+
+	err = os.Setenv("WSHOST", fmt.Sprintf("ws://127.0.0.1:%d", node.WSPort))
+	require.NoError(t, err)
+
+	defer os.Unsetenv("WSHOST")
+
 	parts := strings.Fields(command)
 	data, err := exec.Command(parts[0], parts[1:]...).Output()
-	assert.NoError(t, err, string(data))
 
+	fmt.Println(len(data))
 	//uncomment this to see log results from javascript tests
-	//fmt.Printf("%s\n", data)
+	fmt.Printf("%s\n", data)
+
+	assert.NoError(t, err, string(data))
```

Comment:
Is there a point showing the data if there is an error? I think the error should be enough information right?

```suggestion
	assert.NoError(t, err)
```

Reply:
the data contains information about what's going wrong on javascript test side, the err just shows: `command ... exit with non zero code`